### PR TITLE
Replace lavalamp with jpbetz as sig-apimachinery TL

### PR DESCRIPTION
**Pull request by jpbetz**

### CODE REVIEW

File: sig-api-machinery/README.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ The Technical Leads of the SIG establish new subprojects, decommission existing
 subprojects, and resolve cross-subproject technical issues and decisions.
 
 * David Eads (**[@deads2k](https://github.com/deads2k)**), Red Hat
-* Daniel Smith (**[@lavalamp](https://github.com/lavalamp)**), Google
+* Daniel Smith (**[@jpbetz](https://github.com/jpbetz)**), Google
```
Comment (dims) [inline sig-api-machinery/README.md:36]: @jpbetz there is still "Daniel Smith" on this line :)

Comment (jpbetz) [reply sig-api-machinery/README.md]: yikes! Fixed.

**Comment by mrbobbytables** [comment]
@jpbetz changes should be made in sigs.yaml and then when you run make it'll take care of everything 👍

**Comment by jpbetz** [comment]
> Joe Betz changes should be made in sigs.yaml and then when you run make it'll take care of everything +1

Great. Done!

**Comment by palnabarun** [comment]
Should we remove the hold here since it's been 2 weeks as well?

**Pull request merged**